tests/conftest.py

```diff
@@ -104,8 +103,16 @@ def literal(value: Union[str, List[Any]]) -> str:
         sample_clause = f"SAMPLE {sample}" if sample else ""
         match_clause = f"MATCH ({entity} {sample_clause})"
 
-        selected = ", ".join(map(func, legacy.get("selected_columns", [])))
-        select_clause = f"SELECT {selected}" if selected else ""
+        aggregations = []
+        for a in legacy.get("aggregations", []):
+            if a[0].endswith(")") and not a[1]:
+                aggregations.append(f"{a[0]} AS {a[2]}")
+            else:
+                agg = func(a)
+                aggregations.append(agg)
+
+        expressions = aggregations + list(map(func, legacy.get("selected_columns", [])))
+        select_clause = f"SELECT {', '.join(expressions)}" if expressions else ""
```

Comment:
Did you move the aggregation section in order to ensure expressions are generated in the same order between the legacy query and the snql query.

Reply:
Yes I did.